handlers/missing_test/__init__.py

```diff
@@ -1,20 +1,36 @@
 from eventhandler import EventHandler
 
-reftest_required_msg = 'These commits modify layout code, but no reftests are modified. Please consider adding a reftest!'
+REFTEST_REQUIRED_MSG = 'These commits modify {0} code, but no reftests are modified. Please consider adding a test!'
```

Comment:
Let's call this "TEST_REQUIRED_MSG", and change the remaining "reftests" to "tests".